cli/cmd/cluster/update.go

```diff
@@ -1,34 +1,64 @@
 package cluster
 
 import (
-	"errors"
 	"fmt"
 
 	"github.com/spf13/cobra"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/grtl/mysql-operator/cli/config"
+	"github.com/grtl/mysql-operator/cli/options"
+	"github.com/grtl/mysql-operator/cli/util"
 )
 
-var replicasNumber uint32
+var (
+	portUpd     int32
+	replicasUpd int32
+)
 
 var clusterUpdateCmd = &cobra.Command{
 	Use:   "update [cluster name]",
-	Short: "A short description of cluster update",
-	Long: `A longer description of cluster update with usage:
-msp cluster update "my-cluster" --replicas=4`,
-	Args: func(cmd *cobra.Command, args []string) error {
-		if len(args) < 1 {
-			return errors.New("Cluster name is required")
-		}
-		return nil
-	},
+	Short: "Updates a MySQL Cluster",
+	Long: `Updates a MySQL Cluster.
+Specify replicas number, port or secret name to update your MySQL Cluster:
+msp cluster update "my-cluster" --replicas 4 --port 1337 --secret "new-secret"`,
```

Comment:
Updating the secret is mentioned here, but it doesn't look like any of the following code deals with it.